urls.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
Instead of urls.json, we could also use a commonjs module:

```js
module.exports = []
```

But, I usually prefer json requires like this for static data.